tests/cypress/support/commands.js

```diff
@@ -242,7 +242,11 @@ Cypress.Commands.add('openTaskJob', (taskName, jobID = 0, removeAnnotations = tr
 
 Cypress.Commands.add('interactControlButton', (objectType) => {
     cy.get('body').focus();
-    cy.get(`.cvat-${objectType}-control`).trigger('mouseleave').trigger('mouseout').trigger('mouseover');
+    cy.get(`.cvat-${objectType}-control`).trigger('mouseleave').trigger('mouseout').trigger('mousemove').trigger('mouseover');
+    cy.get(`.cvat-${objectType}-control`).should('have.class', 'ant-popover-open');
```

Comment:
Can we avoid using ``ant-*`` classes?

Reply:
I thought to use this as an additional check. But it works without it. Deleted it.